apps/src/templates/sectionAssessments/assessmentDataShapes.js

```diff
@@ -18,3 +12,29 @@ export const questionDataPropType = PropTypes.shape({
   answers: PropTypes.arrayOf(answerDataPropType),
   notAnswered: PropTypes.number.isRequired,
 });
+
+// Shapes for single student multiple choice tables
+
+// Represents an individual students response to a single
+// multiple choice question.
```

Comment:
Great comments on these PropTypes explaining what they're used for!

Nit: The missing apostrophe sort of changes the meaning. Should be `individual student's response`.  Could also be terser: "Represents one student's response to one multiple choice question."